src/lightmap.h

```diff
@@ -38,7 +38,14 @@ static constexpr float LIGHT_TRANSPARENCY_OPEN_AIR = 0.038376418216f;
 // indicates starting (full) visibility (for seen_cache)
 static constexpr float VISIBILITY_FULL = 1.0f;
 
-#define LIGHT_RANGE(b) static_cast<int>( -std::log(LIGHT_AMBIENT_LOW / static_cast<float>(b)) * (1.0 / LIGHT_TRANSPARENCY_OPEN_AIR) )
+constexpr inline int LIGHT_RANGE( float b )
+{
+    if( b == 0.0f ) {
```

Comment:
```suggestion
    if( b <= LIGHT_AMBIENT_LOW ) {
```
Unless there's some way a negative range would make sense?

Reply:
I am not familiar enough with this code to be comfortable doing more than fixing the obvious UB here, so I'm not particularly comfortable committing that change.

Reply:
Understand! (Who would be most familiar with this code?)

Reply:
Kevin, maybe?

Reply:
Mind if I request him as a reviewer, or should I inquire via discord?

Reply:
I don't mind any method.